File: eR1985/eR1985.php
<?php
$X = $_POST['xvalue'];
$Y = $_POST['yvalue'];
$power = 0;
if($Y >= 0)
{
	$power = $Y;
}
else
{
	$power = -$Y;
}
$Z = 1;
while($power != 0)
{
	$Z = $Z * $X;
	$power = $power - 1;
}
if($Y < 0)
{
	$Z = 1 / $Z;
}
$Z = $Z + 1;

File: eR1985/eR1985.c
int eR1985(int x, int y){
        int power = 0;
        if (x >= 0)
        {
                power = y;
        }
        else
        {
                power = -y;
        }
        int z = 1;
        while (power != 0)
        {
                z = z * x;
                power = power - 1;
        }
        if (y < 0)
        {
                z = 1 / z;
        }
        z = z + 1;
        return z;
}